spec/services/keybase_proof_service_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe KeybaseProofService, type: :service do
+  before do
+    base_url = 'https://keybase.io/_/api/1.0/sig/check_proof.json'
+    params = "domain=mastodon.fake&kb_username=cryptoalice&username=alice&sig_hash=11111111111111111111111111"
```

Comment:
single quotes? to keep it consistent or is that not a rubyism?

Reply:
did this too. you are correct. single quotes whenever possible since they don't do interpolation.